todo_app/data/items.py

```diff
@@ -16,4 +16,25 @@ class Item:
     title: str
     description: str
     status: Status
-    due: Optional[date]
+    last_modification: datetime
+    due: Optional[datetime] = None
+
+
+class ItemView:
+    def __init__(self, items: List[Item]):
+        self.sorted_by_due_items = sorted(items, key=lambda item: (item.due is None, item.due))
+        self.in_progress = [item for item in self.sorted_by_due_items if item.status == Status.IN_PROGRESS]
```

Comment:
Even for something as simple as this, it would be nice to make these a property that freshly performs this filtering so that `sorted_by_due_items` and `in_progress` can't end up out of sync